src-tauri/src/main.rs

```diff
@@ -67,7 +67,7 @@ pub fn main() -> anyhow::Result<()> {
 
     // Build a stdout logger.
     let stderr = ConsoleAppender::builder()
-        .encoder(Box::new(PatternEncoder::new("[{d(%d-%m-%Y %H:%M:%S)}] {l} - {m}\n")))
+        .encoder(Box::new(PatternEncoder::new("[{d(%d-%m-%Y %H:%M:%S)}] | {h({l}):5.5} | {m}\n")))
```

Comment:
h - Highlight in der Log-Level-Farbe

:5.5 -  Log-Level sind immer 5 Zeichen. Wenn es weniger sind wird Leerzeichen eingefügt, wenn es mehr sind es auf 5 gekürzt.
![image](https://github.com/user-attachments/assets/e2b76449-b17a-4a32-930a-3c0d42b952aa)

Reply:
sehr geil?

Reply:
Nicht so geil wie wenn ich das für html geschafft hätte 😔 aber für Bughunting schon.